tests/functional/modules/test_zos_volume_init_func.py

```diff
@@ -100,25 +101,62 @@ def test_index_param(ansible_zos_module, params):
             assert INDEX_CREATION_SUCCESS_MSG not in content_str
 
     # bring volume back online
-    hosts.all.zos_operator(cmd=f"vary {TEST_VOL_ADDR},online")
+    hosts.all.zos_operator(cmd=f"vary {volume_2},online")
+
+
+def test_index_param_index_sms_false(ansible_zos_module, volumes_on_systems):
+    hosts = ansible_zos_module
+
+    volumes = Volume_Handler(volumes_on_systems)
+    volume_1 = volumes.get_available_vol()
+    volume_2 = volumes.get_available_vol()
+
```

Comment:
Same question here: is there any issue if volume_1 == volume_2?  We used to have issues when we only had 1 testable volume.

Reply:
Same as the comment on the top, but also I create a [issue](https://github.com/ansible-collections/ibm_zos_core/issues/1832) for a new tool for this specific test suite